docs/symbols.md

```diff
@@ -4,38 +4,39 @@ List of symbols/abbreviations.
 
 | | |
 |-|-|
-| 3T-SBOM | Tool-to-Tool Software Bill of Material |
+| 3T-SBOM | Tool-to-Tool Software Bill of Materials Exchange |
 | ABNF | Augmented Backus–Naur form |
-| AI | Artificial Intelligence |
+| AI | Artificial intelligence |
 | BNF | Backus–Naur form |
-| BOM | Bill of Material |
-| CISA | Cybersecurity and Information Security Agency |
-| CISQ | Center for Information and Security Quality |
-| CPE | Common Package Enumeration |
-| CVE | Common Vulnerabilies and Exposures |
+| BOM | Bill of materials |
```

Comment:
We need to be consistent... if a letter is in acronym, should be capitalized.     So Bill Of Materials, please